system_tests/arbos_upgrade_test.go

```diff
@@ -64,16 +63,16 @@ func TestScheduleArbosUpgrade(t *testing.T) {
 	}
 }
 
-func checkArbOSVersion(t *testing.T, testClient *TestClient, expectedVersion uint64, scenario string) {
```

Comment:
why did we delete this?

Reply:
I am keeping this test commented out similarly to how it is on integration.